snuba/query/snql/expression_visitor.py

```diff
@@ -79,51 +86,51 @@ def visit_column_name(node: Node, visited_children: Iterable[Any]) -> Column:
 
 
 def visit_low_pri_tuple(
-    node: Node, visited_children: Tuple[LowPriOperator, Expression]
+    node: Node, visited_children: Tuple[LowPriOperator, Any, Expression]
 ) -> LowPriTuple:
-    left, right = visited_children
-
+    if len(visited_children) == 2:
+        left, right = visited_children[0], visited_children[1]
```

Comment:
What causes this case to be hit? Aren't there are always 3 children now?

Reply:
Yeah I'm not sure why I had this check here. I'll remove it.